modules/default/calendar/calendar.js

```diff
@@ -135,14 +136,19 @@ Module.register("calendar", {
 			var event = events[e];
 			var eventWrapper = document.createElement("tr");
 
-			if (this.config.colored) {
+			if (this.config.colored && !this.config.coloredSymbolOnly) {
 				eventWrapper.style.cssText = "color:" + this.colorForUrl(event.url);
 			}
 
 			eventWrapper.className = "normal";
 
 			if (this.config.displaySymbol) {
 				var symbolWrapper = document.createElement("td");
+
+				if (this.config.colored && this.config.coloredSymbolOnly) {
```

Comment:
This should be only `this.config.colored` in your case if `coloredSymbolOnly` is false but colored is true, it will not be colored, but should be

Reply:
The case of `colored` is true and `coloredSymbolOnly` is false is already handled on line 139 and will mark the entire `tr` row with color, which means the symbol `td` cell will also implicitly have color.

This line catches the case where `colored` is true and `coloredSymbolOnly` is also true, and adds the color to just the `td` cell.